pkgs/development/python-modules/django-mfa3/default.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  django,
+  setuptools,
+  pyotp,
+  fido2,
+  qrcode,
+  coverage,
+}:
+
+buildPythonPackage rec {
+  pname = "django-mfa3";
+  version = "0.13.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "xi";
+    repo = "django-mfa3";
+    rev = "refs/tags/${version}";
+    hash = "sha256-O8po7VevqyHlP2isnNnLbpgfs1p4sFezxIZKMTgnwuY=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    django
+    pyotp
+    fido2
+    qrcode
+  ];
+
+  # qrcode 8.0 not supported yet
+  # See https://github.com/xi/django-mfa3/pull/14
+  pythonRelaxDeps = [ "qrcode" ];
+
+  nativeCheckInputs = [ coverage ];
+
+  checkPhase = ''
+    coverage run -m django test --settings tests.settings
```

Comment:
Do we must run coverage? Since we don't really do anything with it, it would be nicer to omit them.

Reply:
Good point, fixed it!